eng/Tools.props

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<!-- All Rights Reserved. Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project>
```

Comment:
Where did this non-standard “All Rights Reserved” license header come from?

Reply:
That was introduced with the arcade bring-up i.e. here: https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Tests.props#L2. Atm we use it in some of our files, i.e. in the arcade corefx specific packages.